backend/common/src/main/java/ai/verta/modeldb/common/db/migration/Migration.java

```diff
@@ -3,25 +3,35 @@
 import lombok.Value;
 
 @Value
-public class Migration implements Comparable<Migration> {
+class Migration implements Comparable<Migration> {
   String filename;
 
   @Override
   public int compareTo(Migration o) {
-    return this.getNumber() - o.getNumber();
+    int versionDiff = this.getNumber() - o.getNumber();
```

Comment:
Are null checks on `o` required here for safety?

Reply:
I *think* I'm ok with things blowing up if somehow a null record got into the set that uses this.  I think if we put explicit null handling into this method, it would just blow up in a more explicit way? Also, I don't think TreeSet allows null values to be put into it, so I _think_ we're safe with assuming no nulls?